nowledge-mem-codex-plugin/hooks/nmem-stop-save.py

```
@@ -343,6 +345,97 @@ def _build_save_command(
+        return
+    encoded = base64.urlsafe_b64encode(
+        json.dumps(safe_payload, separators=(",", ":")).encode("utf-8")
+    ).decode("ascii")
+    command = [
+        sys.executable,
+        str(Path(__file__).resolve()),
+        "--skill-outcome-payload",
+        encoded,
+    ]
+    try:
+        subprocess.Popen(command, **_background_spawn_kwargs())
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)